support-frontend/assets/pages/[countryGroupId]/guardianAdLiteLanding/components/headerCards.tsx

```diff
@@ -87,15 +87,14 @@ export function HeaderCards({
 	const { currencyKey } = getGeoIdConfig(geoId);
 	const currency = currencies[currencyKey];
 	const price =
-		productCatalog.GuardianLight?.ratePlans[contributionType]?.pricing[
+		productCatalog.GuardianAdLite?.ratePlans[contributionType]?.pricing[
 			currencyKey
 		];
 	const formattedPrice = simpleFormatAmount(currency, price ?? 0);
 
 	const guardianAdLiteParams = {
-		product: 'GuardianLight',
+		product: 'GuardianAdLite',
 		ratePlan: contributionType,
-		contribution: price?.toString() ?? '',
```

Comment:
Should we remove showing the contribution in the checkout url?

Reply:
Good question, I meant to flag this. I don't think we use the contribution query string arg anywhere on the Guardian Ad-Lite checkout do we? (Which is why I removed it, I could be missing something though?)